autoscale-marathon-container/pom.xml

```diff
@@ -202,6 +202,14 @@
                     </licenseSets>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>7</source>
+                    <target>7</target>
+                </configuration>
+            </plugin>
```

Comment:
Why are we having to compile with Java 1.7?

Reply:
Actually not required. In this module, for some reason the language level was set to 1.5 by default and looks like IntelliJ added this to fix compilation.